content/source/docs/enterprise/index.html.md

```diff
@@ -17,19 +17,14 @@ For independent teams and small to medium-sized businesses, Terraform Cloud is a
 
```

Comment:
I think the "About Terraform Cloud and Terraform Enterprise" section could use a little reworking. Since this is the Enterprise index page, it might be better to frame the issue more in terms of Enterprise, especially the. second paragraph. Maybe something like "Terraform Enterprise includes the Terraform Cloud application plus other stuff..."

Reply:
Good call! I've merged your suggestion, and we can revisit at a later date if it needs any more adjustments. 👍🏼